Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -1372,7 +1372,7 @@ def get_idp_detections_ids(filter_arg=None, offset: int = 0, limit=INCIDENTS_PER
         :rtype ``dict``
     """
     params = {
-        'sort': 'created_timestamp.asc',
+        'sort': 'start_time.asc',
```

Comment:
so we are intentionally changing this or did you mean to preserve created_timestamp.asc